great_expectations_cloud/agent/event_handler.py

```diff
@@ -75,3 +106,29 @@ def handle_event(self, event: Event, id: str) -> ActionResult:
 
 class UnknownEventError(Exception):
     ...
+
+
+class NoVersionImplementationError(Exception):
+    ...
+
+
+class InvalidVersionError(Exception):
+    ...
+
+
+def _get_major_version(version: str) -> str:
```

Comment:
```suggestion
def _get_major_version(version: str | Version) -> str:
```

Reply:
At this point the version is a string, it's parsed in this function.